how-it-works/00_General/01_overview-of-the-internet-computer.card.md

```diff
@@ -0,0 +1,34 @@
+---
+title: Architecture of the Internet Computer
+---
+
+![](/img/how-it-works/subnet_architecture.png)
+
+# Architecture of the Internet Computer
+
+The Internet Computer implements the vision of a World Computer –  an open and secure blockchain-based network that can host all of humanity's computations in a secure and trustworthy way, witout needing to trust any single party to behave properly.
+To achieve this, the technology behind the Internet Computer (IC) had to be reconceived from ground up, learning many lessons from earlier projects that failed to create a World Computer.
+
+At a high level, the IC's architecture builds on *subnets*, or *subnetworks*: The IC as a whole consists of many subnets, where each subnet is its own blockchain that operates concurrently with and independently of the other subnets.
+A subnet is a blockchain-based *replicated state machine*, that is, a virtual machine that holds state in a secure and non-tamperable manner.
+Each subnet hosts smart contracts, called *canister smart contracts*, or just *canisters*.
+New subnets can be created from nodes added to the IC to scale out the system, analogous to how traditional architectures such as public clouds scale out by adding machines.
+Such a scale-out architecture is unique in the blockchain space and allows for limitless scaling, i.e., combining the security and resiliency properties of blockchains with the scalability properties known from the public cloud.
+
+A canister smart contract can send messages to canisters on the same or different subnets, thereby realizing interactions between canister smart contracts.
+The secure *asynchronous cross-subnet (xnet) communication* between canisters and the resulting loose coupling of subnets is one of the key principles that unlock the scalability of the IC by means of adding new subnets.
+The asynchronous communication between canister smart contracts is also a major difference to most other blockchains in terms of the programming model.
+
+Each subnet is powered by the *core IC protocol* executed by the nodes of the subnet.
+Besides this core protocol, there is much more that is needed to build and run the Internet Computer:
+The *chain-evolution* parts of the IC protocol enable the long-term operation of the system, e.g., by allowing new nodes to securely and efficiently join a subnet or faulty nodes to securely be replaced.
+The IC protocol relies heavily on *chain-key cryptography*, a toolbox of advanced cryptographic mechanisms that power the Internet Computer.
+Governance of the IC is accomplished through a *tokenized Decentralized Autonomous Organization (DAO)*, the *Network Nervous System (NNS)*.
+Each individual dApp on the IC can receive its own governance system similar to the NNS by deploying and customizing a *Service Nervous System (SNS)* for the dApp – without any programming by the dApp developers, but only SNS customization.
```

Comment:
I am more and more inclined to split the page into:

- how the protocol works (description of infrastructure like core protocol, operations, chain-key crypto, ...)
- why a developer should build on the IC

In that perspective, the part about the SNS better fits the dev page.